calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -88,9 +98,32 @@ spec:
   webApplicationFirewall: Enabled
 ```
 
+:::note
+
+WAF can be disabled in your cluster by setting the value of the `webApplicationFirewall` field to `Disabled`.
```

Comment:
I would show an example of the yaml here with the field set to Disabled, I'm not sure some users will know they're meant to be changing the `ApplicationLayer` CR.

Also, the bug [here](https://tigera.atlassian.net/browse/RS-704) says that a brief description of how to disable using the UI would be handy, because it's not completely obvious how to do it.